Gems/ROS2/Code/Source/RobotImporter/URDF/URDFPrefabMaker.cpp

```diff
@@ -465,10 +470,15 @@ namespace ROS2
         }
 
         // Use the first entity based on a link that is not parented to any other link
-        if (!linkEntityIdsWithoutParent.empty() && linkEntityIdsWithoutParent.front().IsValid())
+        if (!linkEntityIdsWithoutParent.empty())
```

Comment:
This comment is now out of date.

All of the links which aren't a child of another link would have a control plugin added to it.

Reply:
Fixed.